### PR TITLE
Reorganize cloudservices sites in runner.yaml

### PR DESCRIPTION
**Pull request by jvehent**

### CODE REVIEW

File: conf/runner.yaml [hunk 1]
```diff
@@ -135,6 +135,69 @@ runs:
...
+        #AMO
+        - addon.mozilla.org
+        - addons.mozilla.org
+        - blocklist.addons.mozilla.org
+        - builder.addons.mozilla.org
+        - services.addons.mozilla.org
+        - static.addons.mozilla.net
+        - versioncheck-bg.addons.mozilla.org
+        - versioncheck.addons.mozilla.org
+        - compatibility-lookup.services.mozilla.com
```
Comment (jvehent) [inline conf/runner.yaml]: @jasonthomas : All the sites above use intermediate TLS. This change will alert us if the configuration changes. r?

File: conf/runner.yaml [hunk 2]
```diff
@@ -135,6 +135,69 @@ runs:
...
+
+        # Balrog
+        - aus5.mozilla.org
```
Comment (jvehent) [inline conf/runner.yaml]: @mostlygeek : AUS use intermediate TLS. This change will alert us if the configuration changes. r?

File: conf/runner.yaml [hunk 3]
```diff
@@ -135,6 +135,69 @@ runs:
...
+
+        # Firefox Settings (Kinto)
+        - blocklists.settings.services.mozilla.com
+        - firefox.settings.services.mozilla.com
+        - webextensions.settings.services.mozilla.com
```
Comment (jvehent) [inline conf/runner.yaml]: @Micheletto : All the sites above use intermediate TLS. This change will alert us if the configuration changes. r?

File: conf/runner.yaml [hunk 4]
```diff
@@ -135,6 +135,69 @@ runs:
...
+        # Test Pilot
+        - testpilot.firefox.com
+        - embedly-proxy.services.mozilla.com
+        - universalsearch.testpilot.firefox.com
+        
+        # Miracle (Context Graph collection endpoint)
+        - miracle.services.mozilla.com
+        
+        # Pageshot
+        - pageshot.net
```
Comment (jvehent) [inline conf/runner.yaml]: @relud: All the sites above use intermediate TLS. This change will alert us if the configuration changes. r?

Comment (relud) [inline conf/runner.yaml]: I don't recognize miracle.services.mozilla.com, but the rest use the aws elb policy, so r+ if that meets the intermediate tls rules.

File: conf/runner.yaml (hunk 2 above)
Comment (mostlygeek) [inline conf/runner.yaml]: R+

**Pull request merged**